e2e/utils.go

```diff
@@ -585,6 +585,23 @@ func validatePVCClone(totalCount int, sourcePvcPath, sourceAppPath, clonePvcPath
 				LabelSelector: fmt.Sprintf("%s=%s", appKey, label[appKey]),
 			}
 			wgErrs[n] = createPVCAndApp(name, f, &p, &a, deployTimeout)
+			if wgErrs[n] == nil && validateEncryption {
+				if kmsIsVault(kms) || kms == vaultTokens {
+					imageData, sErr := getImageInfoFromPVC(p.Namespace, name, f)
+					if sErr != nil {
+						wgErrs[n] = fmt.Errorf("failed to get image info for %s namespace=%s volumehandle=%s error=%w", name, p.Namespace, imageData.csiVolumeHandle, sErr)
+					} else {
+						// check new passphrase created
+						stdOut, stdErr := readVaultSecret(imageData.csiVolumeHandle, kmsIsVault(kms), f)
+						if stdOut != "" {
+							e2elog.Logf("succeesfully read the passphrase from vault: %s", stdOut)
```

Comment:
Typo `succeesfully`. But I guess you should rather check for `sdtOut == ""` and log an error in that case?

Reply:
yeah sure

Reply:
for logging the error, we have stdErr != ""..  stdOut is just to make sure in that the test is running fine.